pkg/manifest/os.go

```diff
@@ -400,6 +400,13 @@ func (p *OS) getBuildPackages(distro Distro) []string {
 		packages = append(packages, tomlPkgsFor(distro)...)
 	}
 
+	if p.platform.GetBootloader() == platform.BOOTLOADER_UKI {
+		// Only required if the hmac stage is added, which depends on the
+		// version of uki-direct. Add it conditioned just on the bootloader
+		// type for now, until we find a better way to decide.
+		packages = append(packages, "libkcapi-hmaccalc")
+	}
```

Comment:
I have a wild idea: If we return the results of each transaction depsolving, in addition to the complete result (which we already discussed at some point), we could depsolve packages such as this one in a separate transaction. We would opportunistically depsolve additional options and include the additional package set in the one to be installed conditionally here.

Reply:
Interesting.  Yeah, something to think about.